core/google/api/core/exceptions.py

```diff
@@ -332,7 +332,7 @@ def exception_class_for_http_status(status_code):
         status_code (int): The HTTP status code.
 
     Returns:
-        type: the appropriate subclass of :class:`GoogleAPICallError`.
+        :func:`type`: the appropriate subclass of :class:`GoogleAPICallError`.
```

Comment:
This doesn't make sense?

Reply:
It's to help sphinx disambiguate between multiple targets. It considers builtin `type` to be a function.